src/node_crypto.cc

```diff
@@ -121,6 +121,7 @@ const char* const root_certs[] = {
 };
 
 X509_STORE* root_cert_store;
+std::vector<X509 *>* root_certs_vector;
```

Comment:
May I ask you to make it `X509*` here? We generally try to keep asterisk with the type name.